query-node/mappings/src/content/video.ts

```diff
@@ -465,6 +467,14 @@ function integrateVideoMediaMetadata(
   // connect encoding to media metadata object
   mediaMetadata.encoding = encoding
 
+  // ensure predictable ids
+  if (!mediaMetadata.encoding.id) {
```

Comment:
Does this test really work? What if the id is `0`

Reply:
Yes this could cause a trouble in a rare case. It has been prevented by setting mining `getNextId` result to `1`.